dynawo/cmake/TestCoverage.cmake

```diff
@@ -13,7 +13,7 @@ if (NOT (CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "TestCov
 endif()
 
 # Compilation flags
-set(COVERAGE_CXX_FLAGS "-g -O0 -fprofile-arcs -ftest-coverage")
+set(COVERAGE_CXX_FLAGS "-g -O1 -fprofile-arcs -ftest-coverage -fno-omit-frame-pointer -fno-optimize-sibling-calls")
```

Comment:
why this change ?

Reply:
Should improve the issue with the coverage (prevent from creating branches for every possibility of a if)